code/modules/client/preferences.dm

```diff
@@ -208,6 +208,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	dat += "<a href='?_src_=prefs;preference=randomize_name'>(R)</a>"
 	dat += "<br>"
 	dat += "Always Pick Random Name: <a href='?_src_=prefs;preference=random_name'>[random_name ? "Yes" : "No"]</a>"
+	dat += "<br><b>Citizenship</b>: <a href = '?_src_=prefs;preference=citizenship'>[citizenship]</a><br/>"
```

Comment:
Letting players pick the side their on is a bad idea. I recommend autoabalancing the client to one side or the other side when they connect.

Reply:
I would imagine this being more of a 'preference' for the player. Autobalance should indeed still happen.